shared/util/errors.js

```diff
@@ -118,6 +118,8 @@ export const niceError = (e: RPCError) => {
       return 'Looks like the internet is gone...'
     case RPCTypes.constantsStatusCode.scbadsignupusernamedeleted:
       return 'Looks like this user was deleted, or something'
+    case RPCTypes.constantsStatusCode.scstreameof:
+      return 'Looks like we took too long. Try again, but a little bit quicker maybe'
```

Comment:
Does EOF always mean time out though?

Also, 1) in this case user would need to generate another QR code; 2) for me there wasn't a way to go back, and I had to kill the app.

Reply:
on error provision has to be canceled and restarted